### DISCUSSION TITLE
What "apart from" means

### DISCUSSION THREAD

**Discussion by tmakarios** [Freely-Given-org/OpenEnglishTranslation--OET#5] [General]
I think my default interpretation of "apart from" is roughly equivalent to "except", but this gives a kind of pantheistic meaning to the second half of the OET-RV's Yohan 1:3, where it says "not even one thing came into existence apart from him".

Is there a reason "apart from" was chosen here, rather than "without", as in the OET-LV, or "except through", or something like that?

**Comment by RobH123** [comment]
Yes, thanks. Reworked in commit 07f638f
